.github/workflows/component.yml

```diff
@@ -77,4 +77,7 @@ jobs:
         run: npm run coverage 
       - name: Upload to Codecov 
         if: matrix.os == 'ubuntu-latest' #only upload to codecov on ubuntu VM 
-        run: bash <(curl -s https://codecov.io/bash)
+        uses: codecov/codecov-action@v3
+        with:
+          directory: ${{ github.workspace }}/build/reports/jacoco/jacocoReport
+          files: jacocoReport.xml
```

Comment:
This only uploads the backend coverage report. There is a frontend coverage report that also needs to be uploaded.